PhysicsTools/NanoAOD/plugins/GenWeightsTableProducer.cc

```diff
@@ -276,6 +277,25 @@ class GenWeightsTableProducer : public edm::global::EDProducer<edm::StreamCache<
                                         break;
                                     }
                                 }
+                            } else if (groups.str(2) == "NNPDF30_lo_as_0130.LHgrid") {
```

Comment:
Just noticing that the corresponding updates in the master have comments added, see
https://github.com/cms-sw/cmssw/pull/21300/files#diff-5e612deb23d740e47f5b7ede04e2b5bbR280
(not an issue, anyhow)

Reply:
given that it is ok in the master and is just a set of comments I'd leave it as it is (if I cherry pick the master commit  on 94Xbranch I'd change the order of commits and I'd like to avoid any more troubles with mergeability and rebases)